### ISSUE TITLE
Add github actions workflows for CI and publish

### ISSUE THREAD

**Issue by shubhamzanwar** [flipspace/flippie#20]

**Comment by hussamkhatib** [comment]
hey maintainer ! can you assign this issue to me 
just to confirm does "Add github actions workflows for CI and publish" means auto deploy website on a push in the repo in the branch gh pages?

**Comment by shubhamzanwar** [comment]
Yes, we could start with that. When a new commit is pushed, we can redeploy the website
Eventually, we should also add a workflow to publish the package to npm registry when a new tag is pushed

**Comment by hussamkhatib** [comment]
cool i will start with website thing first.

**Comment by hussamkhatib** [comment]
I think this won't be required for website as it is deployed with github pages. 
Changes will made on push automatically.

**Comment by shubhamzanwar** [comment]
You would still need to deploy the website by running `yarn deploy` inside the website folder.
Instead of doing it manually, we could write a github action to do the same on every commit push to master.

What do you think?

**Comment by hussamkhatib** [comment]
sure.
i dint realize there was a website folder in master branch.
I thought everything was served in gh-pages branch.

**Comment by hussamkhatib** [comment]
i could find this [workflow](https://github.com/jeoy/github-deploy-actions]) from github action.
I am still not sure if i am doing it right ,this is what i have tried ⬇️.  
![image](https://user-images.githubusercontent.com/52914487/95014797-eb5b6b80-0666-11eb-8fd5-e5574bdc99b7.PNG)
